chirp/wxui/main.py

```diff
@@ -1809,14 +1809,27 @@ def _menu_reload_driver(self, event, andfile=False):
 
     def _menu_interact_driver(self, event):
         LOG.warning('Going to interact with radio at the console')
+
+        main = self
```

Comment:
Are you doing this because the `embed()` call automatically imports locals from the calling scope or something? Are you not able to pass a subset to make it equivalent? I'm specifically passing locals (instead of `locals()`) to keep this restricted to specific uses so it would be preferred to keep it equivalent.